packages/eslint-plugin/src/util/getWrappingFixer.ts

```diff
@@ -205,3 +215,21 @@ function isLeftHandSide(node: TSESTree.Node): boolean {
 
   return false;
 }
+
+/**
+ * Checks if a node's parent is arrow function expression and a inner node is object expression
+ */
+function isObjectExpressionInOneLineReturn(
+  node: TSESTree.Node,
+  innerNode: TSESTree.Node,
+): boolean {
+  if (
+    node.parent?.type === AST_NODE_TYPES.ArrowFunctionExpression &&
+    node.parent?.body === node &&
```

Comment:
Btw, I see the test coverage is dropping on this line... I _suspect_ it's fine, as a general-purpose thing. But it'd be nice not to have to run unused code. 🤔 

(delete the `node.parent?.type` line -> no tests fail)

Reply:
Oh I missed that case 😅 I added it!
I also want to add a case for `node.parent.body` line, but I cannot come up with...